programs/stake/src/stake_state.rs

```diff
@@ -560,7 +560,11 @@ impl Stake {
     ) -> (u128, u64) {
         // if there is no newer credits since observed, return no point
         if new_vote_state.credits() <= self.credits_observed {
-            return (0, 0);
+            if fix_stake_deactivate {
```

Comment:
For testnet, we can freeride on flag (backed by already activated `stake_program_v2`) *only after inflation is started*.
For mainnet-beta, we are still safe to edit this.

Reply:
Strictly speaking, we don't need this `fix_stake_deactivate` flag at all. That's because we can prove we never use the second item in this returned tuple statically. Any I'm adding this for _consistency_ and _just be in the safe side_.